### ISSUE TITLE
verbs 'restore'  and 'recycle' do not work?

### ISSUE THREAD

**Issue by zpydee** [nlp-compromise/compromise#412]
in an angular project:

```
import * as compromise from 'compromise';
console.log(compromise('add').verbs().toPastTense().out());
```

works, but both:

```
import * as compromise from 'compromise';
console.log(compromise('restore').verbs().toPastTense().out());
```

and

```
import * as compromise from 'compromise';
console.log(compromise('recycle').verbs().toPastTense().out());
```

do not.

any suggestions?

**Comment by zpydee** [comment]
there seem to be a problem with verbs starting with `re`

**Comment by spencermountain** [comment]
hey Dave, yeah sorry this isn't very clear at all.
restore and recycle are both being (mistakenly) tagged as a noun.

to remedy this, you can co-erce them to be a verb, first, like this:
https://runkit.com/spencermountain/59b6b622b94eaf001183d6b5

this is pretty weird, i acknowledge.

**Comment by rtsao** [comment]
> there seems to be a problem with verbs starting with re

Indeed, these verbs are also tagged as nouns:
- Refactor
- Release
- Revert

**Comment by spencermountain** [comment]
yeah, `re-` is pretty strong, but [re](https://www.morewords.com/starts-with/re/) is not as strong a signal as i had earlier hoped, i think.
i think, since we have our lexicon compression thing working by prefix, it's basically free to add these to our lexicon. 

will do in v11,
got any more?

**Comment by rtsao** [comment]
@spencermountain these also seem to fail:

- repeat
- rebuild
- reconcile
- record
- redefine
- recover
- restructure
- resolve

**Comment by spencermountain** [comment]
hey, haven't gotten to this yet, but as of v11, there's support in the api for defining irregular (or in this case, mistaken) conjugations:
https://runkit.com/spencermountain/59f34c4fce71dc00133b8eee
cheers

**Comment by rtsao** [comment]
Found another verb: 'expose' isn't being categorized correctly either

**Comment by rtsao** [comment]
Another verb: 'upgrade'

**Comment by KevinGrandon** [comment]
Another verb: 'introduce'

**Comment by spencermountain** [comment]
hey, i think I got all of the issues mentioned on this ticket today,
it is now published in `11.2.3`
please let me know if I missed any
thanks for the help.

**Comment by KevinGrandon** [comment]
Thank you for the fix. I've filed a new ticket with another batch here whenever someone has time: https://github.com/nlp-compromise/compromise/issues/448
